utils.sh

```diff
@@ -238,7 +243,24 @@ function generate_templates {
     fi
 
     # clouds.yaml
-    go run metal3-templater.go clouds.yaml.template "$CLUSTER_PRO_IF" "$PROVISIONING_NETWORK" "$MACHINE_OS_IMAGE_URL" "$BOOTSTRAP_PROVISIONING_IP" "$CLUSTER_PROVISIONING_IP" > clouds.yaml
+    if [[ "$OPENSHIFT_VERSION" != "4.6" ]]; then
```

Comment:
This is going to break as soon as 4.7 opens.

Reply:
Fixed